Dockerfile.template

```diff
@@ -85,10 +86,28 @@ RUN sudo apt-get update && sudo apt-get install -y \
 	php -m | grep "gd" && \
 	php -m | grep "sockets"
```

Comment:
@JalexChen was working on verifying this works on my end with our use case but hit snags in the verification steps here. I'm not seeing `gd` or `sockets` available in the modules for the compiled PHP 7.2.34. If you've got tips on how to get past that happy to keep working on this, otherwise I'll wait 😄

Reply:
@JalexChen sadly this client is still tied to PHP 7.2 due to some other dependencies. Working on getting them unblocked there but it's currently a hard requirement. If it's better for you to focus on 7.4+ and we stay on the older php CI images for now, completely understood.